### DISCUSSION TITLE
Creator Category & Badges

### DISCUSSION THREAD

**Discussion by jsonpreet** [deso-protocol/dips#99] [Medium CIPs]
These suggestions by https://bitclout.com/u/GraphicShivam

I had some suggestions regarding few changes in Bitclout frontend

1. Like Instagram, we can add category feature in creators profile. Like If  I am an artist or a singer or comedian or anything, so I can add that feature to my profile and when some clicks at that tag/category, they will be directed to whole new page/website where the creators of same category will be sorted and hence easier for public/investors to get to them.

2. We can do this same as by giving badges to them next to their profile
Like an artist will have his artist badge next to his  profile, so if anyone visits the profile for the first time, they can easily find that which type of account is this.

Also we can upgrade the level of badges based on the activity of the profile, like level 1, level 2
And if some has rugged a profile, we can give them the rugged badge or something like that

**Comment by arhebbar** [comment]
Not to plug in our project. But, check out creatiers.co and @CreaTiers on BitClout. That's what we are working on and over time, we hope to commit back to core if there's interest.
